frontend/app/src/components/accounts/AccountBalanceTable.vue

```diff
@@ -95,26 +95,24 @@
           <template v-if="!loopring">
             <account-asset-balances
               :title="$t('account_balance_table.assets')"
-              :assets="accountAssets(item.address)"
+              :assets="accountAssets(item.address).value"
```

Comment:
is there not a better way to do this with composition other than calling `.value`?

Reply:
I can't find/think of other better ways.
The only thing that makes sense is to map the `accountAssets` for all data inside script, so inside the template, the function doesn't need to be called

Reply:
ok, this is something for the future then.